wrapper.html

```diff
@@ -166,46 +175,42 @@
 	<link rel='stylesheet' id='baseline-rtl-css' href='https://dbqvwi2zcv14h.cloudfront.net/ak/ak-v3-rtl.css'
 		type='text/css' media='all' />
 	{% endif %}
-
+	<!-- If custom WebP URL is set, use WebP with the page_background_image as a fallback. Else just use page_background_image. -->
+	{% if page.custom_fields.page_background_image %}
 	<style>
-		{
-			% if page.custom_fields.page_background_image %
-		}
-
-		#body-mobile-background {
-			background-color: {
-					{
-					templateset.custom_fields.color_page
+		{% if page.custom_fields.page_background_image_webp_url %}
+			@supports not (background-image: url('{{ page.custom_fields.page_background_image_webp_url }}')) {
+				#body-mobile-background {
+					background-image: url('{{ page.custom_fields.page_background_image }}');
 				}
 			}
-
-			;
-			background-image:url({{ page.custom_fields.page_background_image }});
-		}
-
-			{
-			% endif %
-		}
+			@supports (background-image: url('{{ page.custom_fields.page_background_image_webp_url }}')) {
+				#body-mobile-background {
+					background-image: url('{{ page.custom_fields.page_background_image_webp_url }}');
+				}
+			}
+		{% else %}
+			#body-mobile-background {
+				background-color: '{{ templateset.custom_fields.color_page }}';
+				background-image: url('{{ page.custom_fields.page_background_image }}');
+			}
+		{% endif %}
 	</style>
-
+	{% endif %}
 	{% block css_additions %}{% endblock %}
-	<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
-	<script
-  src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"
-  integrity="sha256-VazP97ZCwtekAsvgPBSUwPFKdrwD3unUfSGVYrahUqU="
-  crossorigin="anonymous"></script>
+
 	<script type="text/javascript" src="https://act.350.org/resources/actionkit.js"></script>
-		<!--Recaptcha-->
-	   <script src="https://www.google.com/recaptcha/api.js" async defer></script>
+	{% if page.type != "Donation" %}

```

Comment:
Not necessary, but I wonder if we could consolidate all of these to one if clause rather than multiple.

Reply:
definitely possible, I was just a bit hesitant to mess with the order of the imports

Reply:
Can leave that for another day :laughing: